src/components/TeamMemberTasks/ReviewButton.jsx

```diff
@@ -106,7 +106,7 @@ const ReviewButton = ({ user, task, updateTask }) => {
   };
 
   const handleLink = e => {
-    const url = e.target.value;
+    const url = e.target.value.trim();
```

Comment:
`function sanitizeUrl(raw) {
  const input = raw.trim();
  if (/\s/.test(input)) return null;
  try {
    const url = new URL(input.startsWith('http') ? input : `https://${input}`);
    // Only allow http/https
    if (!['http:', 'https:'].includes(url.protocol)) return null;
    return url.href;
  } catch (e) {
    return null;
  }
}`

Reply:
To protect XSS attacks